cli/src/main/kotlin/commands/UploadResultToPostgresCommand.kt

```diff
@@ -98,7 +100,7 @@ class UploadResultToPostgresCommand : CliktCommand(
         createConnection(postgresConfig).use { connection ->
             val query = "INSERT INTO ${postgresConfig.schema}.$tableName ($columnName) VALUES (to_json(?::json)::jsonb)"
 
-            val json = jsonMapper.writeValueAsString(ortResult).escapeNull()
+            val json = jsonMapper.writeValueAsString(ortResult.normalizeDependencies()).escapeNull()
```

Comment:
Hmm, I'm not sure about the `normalizeDependencies()` name. Maybe better something like `convertGraphToScopes()`?

Reply:
Agree with this comment. Which format is considered as normalized may change in future.